tests/components/tuya/__init__.py

```diff
@@ -173,6 +173,7 @@
     "mcs_oxslv1c9",  # https://github.com/home-assistant/core/issues/139966
     "mcs_qxu3flpqjsc1kqu3",  #  https://github.com/home-assistant/core/issues/141278
     "msp_3ddulzljdjjwkhoy",  # https://github.com/orgs/home-assistant/discussions/262
+    "msp_mzuro9jgjs7uiryn",  # Poopy Nano 2 support
```

Comment:
We link the issue/discussion/PR to make it easier to find the original discussion
```suggestion
    "msp_mzuro9jgjs7uiryn",  # https://github.com/home-assistant/core/pull/155964
```